src/test/crush/CrushWrapper.cc

```diff
@@ -174,6 +174,9 @@ TEST(CrushWrapper, swap_bucket) {
   ASSERT_EQ(1, c->get_bucket_item(a, 1));
   ASSERT_EQ(2, c->get_bucket_item(a, 2));
   ASSERT_EQ(3, c->get_bucket_item(b, 0));
+	
+  // check if it can swap father and son
```

Comment:
please be consistent with the notation in crush. i.e. s/father and son/parent and child/